### ISSUE TITLE
Engine crashes after trying to upload a video

### ISSUE THREAD

**Issue by Nickykun** [turanszkij/WickedEngine#840]
![image](https://github.com/turanszkij/WickedEngine/assets/46878391/18280402-db14-403d-aa61-a626de23f426)

**Comment by turanszkij** [comment]
Yeah, I know this, video only works in Vulkan currently. With DX12 it will either crash or display some corrupted result. For the time being I've given up on it, because I couldn't solve it. Everything from the video is parsed the same way as in vulkan, but somehow DX12 doesn't accept it.

**Comment by turanszkij** [comment]
Video now works with DX12.
